### PR TITLE
Add KangarooSection

### PR DESCRIPTION
**Pull request by mcellier**
Allow to display a margin-less & padding-less full-width section on small devices. We'll use it for several flows in kairos: booking request, profile picture modification...

Storybook demo:
ezgif-3-6ae76f3906c1

Booking request demo:
ezgif-3-ff359d9890e0

### CODE REVIEW

File: src/layout/section/kangarooSection/kangarooSection.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import cc from 'classcat'
+
+export interface KangarooSectionProps {
+  readonly className?: Classcat.Class
+  readonly role?: string
+  readonly children: JSX.Element
```
Comment (wakooka) [inline src/layout/section/kangarooSection/kangarooSection.tsx]: I got a feedback on a previous PR that we should use `React.ReactNode` instead cc @grosbouddha

File: src/layout/section/kangarooSection/kangarooSection.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+/**
+ * The kangaroo section: Renders a fullscreen without margin div on small devices
```
Comment (wakooka) [inline src/layout/section/kangarooSection/kangarooSection.tsx]:
```suggestion
 * The kangaroo section: Renders a div without margins on small devices
```

File: src/layout/section/kangarooSection/kangarooSection.tsx (hunk 1 above)
Comment (grosbouddha) [reply src/layout/section/kangarooSection/kangarooSection.tsx]: React.ReactNode should work with string, JSX and array of JSX and is the most flexible.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[NEW]** Add `KangarooSection`, a fullscreen without spacing on small devices section
```
Comment (grosbouddha) [inline CHANGELOG.md]: Do we need a better name :) ?

Comment (grosbouddha) [reply CHANGELOG.md]: The PR looks good technically, let's have a quick poll at next standup/weekly ?
We can move forward with this name maybe in the meantime.
My real concern is that this one time joke turns into a "I ve no name, let's use a funny name" practice.
Maybe @pith has some good naming inspiration on this one ?

Comment (pith) [reply CHANGELOG.md]: `MediaSection` ?

File: src/layout/section/kangarooSection/kangarooSection.tsx (hunk 1 above)
Comment (mcellier) [reply src/layout/section/kangarooSection/kangarooSection.tsx]: Ok, i'll change that

File: CHANGELOG.md (hunk 3 above)
Comment (mcellier) [reply CHANGELOG.md]: We didn't wanted to put the `Mobile` name in its as it's more tied on a breakpoint than a device. `MapSection` isn't quite good too as it'll be used to the profile picture modification too. We already have a `MediaContentSection`. I don't know what to do :'(

File: src/layout/section/kangarooSection/kangarooSection.tsx (hunk 2 above)
Comment (mcellier) [reply src/layout/section/kangarooSection/kangarooSection.tsx]: I updated

File: src/layout/section/mediaSection/mediaSection.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import cc from 'classcat'
+
+export interface MediaSectionProps {
+  readonly className?: Classcat.Class
+  readonly role?: string
+  readonly children: React.ReactNode
+}
+
+/**
+ * The kangaroo section: Renders a fullscreen div without margins on small devices
```
Comment (wakooka) [inline src/layout/section/mediaSection/mediaSection.tsx]: Update comment 🦘

Comment (mcellier) [reply src/layout/section/mediaSection/mediaSection.tsx]: There were 2 other remnants :x

**Pull request merged**